docs/components/Navigation.js

```diff
@@ -2,74 +2,66 @@
 import { type Node } from 'react';
 import { Box, SideNavigation } from 'gestalt';
 import { useRouter } from 'next/router';
-import sidebarIndex from './sidebarIndex.js';
+import newSidebarIndex, { type siteIndexType } from './siteIndex.js';
```

Comment:
Mian changes here to render the side nav with new index and add platform switcher